.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ references:
 jobs:
   build:
     docker:
-      - image: circleci/buildpack-deps:18.10-scm
+      - image: circleci/buildpack-deps:18.04-scm
```

Comment:
Was bumping the minor version from .10 to .04 intentional?

Reply:
Upgraded to the new cimg/base:current-22.04